src/lib/mongoBackend/mongoListSubscriptions.cpp

```diff
@@ -23,42 +23,171 @@
 * Author: Orion dev team
 */
 
+#include "common/sem.h"
+
+#include "logMsg/logMsg.h"
+#include "logMsg/traceLevels.h"
+
 #include "mongoBackend/mongoListSubscriptions.h"
+#include "mongoBackend/MongoGlobal.h"
+#include "mongoBackend/collectionOperations.h"
+
+#include "mongo/client/dbclient.h"
+
+using namespace ngsiv2;
+
+/* ****************************************************************************
+*
+* setSubscriptionId -
+*/
+extern void setSubscriptionId(Subscription* s, const BSONObj& r)
+{
+  s->id = r.getField("_id").OID().toString();
```

Comment:
Doubt. "_id" must always be there, right?
I mean, if "_id" is not found, what would happen?  SIGSEGV?